src/orb.yml.hbs

```diff
@@ -94,6 +94,57 @@ examples:
           jobs:
             - verify-deployment
 
+  run-task-fargate:
+    description: Start the run of an ECS task on Fargate.
+    usage:
+      version: 2.1
+
+      orbs:
+        aws-cli: circleci/aws-cli@0.1.4
+        aws-ecs: circleci/aws-ecs@x.y.z
+
+      jobs:
+        run-task:
+          docker:
+            - image: circleci/python:3.7.1
+          steps:
+            - aws-ecs/run-task:
+                cluster: "cluster1"
+                task-definition: "myapp"
+                # e.g. "subnet-70faa93b,subnet-bcc54b93"
+                subnet-ids: $SUBNET_ONE, $SUBNET_TWO
+                # e.g. "sg-010a460f7f442fa75"
+                security-group-ids: $SECURITY_GROUP_IDS
+
+      workflows:
+        run-task:
+          jobs:
+            - run-task
+
+  run-task-ec2:
+    description: Start the run of an ECS task on EC2.
+    usage:
+      version: 2.1
+
+      orbs:
+        aws-cli: circleci/aws-cli@0.1.4
+        aws-ecs: circleci/aws-ecs@x.y.z
+
+      jobs:
+        run-task:
+          docker:
+            - image: circleci/python:3.7.1
+          steps:
+            - aws-ecs/run-task:
+                cluster: "cluster1"
+                task-definition: "myapp"
+                aws_vpc: false
```

Comment:
Should this be `awsvpc`?

Reply:
Hey @lokst good catch! I just updated the PR.